webapp-django/crashstats/settings/base.py

```diff
@@ -125,13 +127,20 @@ def path(*dirs):
     'crashstats.tokens.middleware.APIAuthenticationMiddleware',
     'crashstats.crashstats.middleware.Pretty400Errors',
 )
+MIDDLEWARE = list(MIDDLEWARE)
```

Comment:
We can just declare `MIDDLEWARE` as a list from the start instead of converting it.